### ISSUE TITLE
No longer able to modify device via PRTGAPI cmdlets

### ISSUE THREAD

**Issue by sysadmin24** [lordmilko/PrtgAPI#288]
### Describe the bug

Hi,

Not sure when this issue first began. But we've recently updated PRTG as well as migrated PRTG from Server 2012 to Server 2022.

The Core and Probe servers are all Server 2022 now. Have .NET 4.8 installed.
PRTG server was also recently updated as well to the latest version on Passler's site.

Different user accounts have the same issue and from the local core server as well as from remote this issue follows.

We can Connect-PrtgServer fine and can 
Get-Group 
Add-Device

, but once we try 
Get-Device $Device | Get-SensorTarget -RawType snmpdiskfree or 
New-sensorparameters -rawtype Ping
the script errors.

Example: "Get-SensorTarget : Could not authenticate to PRTG; the specified username and password were invalid.
At line:44 char:36
+ ... = Get-Device $Device | Get-SensorTarget -RawType snmpdiskfree "

Seems like once we're trying to modify an object it logs us out.

Any ideas here by chance?
Happening to anyone else?

Thank you!

### Steps to reproduce

```powershell
Different users attempted.
Different source machines (multiple remote and local core server)
Tried updating PRTGAPI module from .14/.15 to .16
```

### What is the output of 'Get-PrtgClient -Diagnostic'?

```powershell
PSVersion      : 5.1.19041.1682
PSEdition      : Desktop
OS             : Microsoft Windows 10 Enterprise
PrtgAPIVersion : 0.9.16
Culture        : en-US
CLRVersion     : 528372
PrtgVersion    : 22.2.76.1705
PrtgLanguage   : english.lng
```

**Comment by sysadmin24** [comment]
Nevermind , helps if I read :). I see the latest version is .18 and once I updated to it everything works again now. Thank you so much!

**Comment by lordmilko** [comment]
Hi @sysadmin24,

The latest version is 0.9.17 (0.9.18 is in pre-release) however as you've surmised this is a duplicate of #268 which was resolved in 0.9.17
